packages/csharp/PortingAssistant/PortingAssistant.Api/Program.cs

```diff
@@ -56,6 +58,10 @@ static void Main(string[] args)
             configuration.DataStoreSettings.S3Endpoint = portingAssistantPortingConfiguration.PortingAssistantConfiguration.DataStoreSettings.S3Endpoint;
             configuration.DataStoreSettings.GitHubEndpoint = portingAssistantPortingConfiguration.PortingAssistantConfiguration.DataStoreSettings.GitHubEndpoint;
 
+            string metricsFolder = Path.Combine(args[2], "logs");
+            string metricsFilePath = Path.Combine(metricsFolder, $"portingAssistant-telemetry-{DateTime.Today.ToString("yyyyMMdd")}.metrics");

```

Comment:
Are metrics for the same day all added to one file?

Reply:
Yes